app/matchers.js

```diff
@@ -0,0 +1,23 @@
+export function transferMatcher(text) {
+  return text.match(/\/transfer ([a-zA-Z\d-]+)/);
+}
+
+export function closeMatcher(text) {
+  return text.match(/\/close ?(not-planned)?/);
```

Comment:
```suggestion
  return text.match(/\/close( not-planned)?/);
```
It shouldn't match `/closenot-planned`

Reply:
sure, not quite right though as that will capture the space in the capturing group.

Reply:
resolved